src/modules/m_sqloper.cpp

```diff
@@ -19,46 +20,83 @@
 
 #include "inspircd.h"
 #include "modules/sql.h"
-#include "modules/hash.h"
 
-class OpMeQuery : public SQLQuery
+class OpGetQuery : public SQLQuery
 {
  public:
-	const std::string uid, username, password;
-	OpMeQuery(Module* me, const std::string& u, const std::string& un, const std::string& pw)
-		: SQLQuery(me), uid(u), username(un), password(pw)
+	bool OnUserCommand; //Record whether or not the OpGetQuery (AKA reloading the list of SQLopers) is done as a result of /OPER, or something like a /rehash or module reload
```

Comment:
Is this necessary? If there is no uid you can just not pass one and check if `uid.empty()`.